Demo Maze

Connagh-Lincoln opened the pull request:
Maze with walls using 3d objects built for top down, Y co-ords set to #0 so should prevent player going through - need to apply collisions. NTS David for player script

Commit:
Demo Maze

Maze with walls using 3d objects, Y co-ords set to #0 so should prevent player. Though still learning about collisions etc. NTS talk to David about player script

ghost commented:
Also, added [ProBuilder](https://assetstore.unity.com/packages/tools/modeling/probuilder-2-x-111418) Plugin in this commit as was originally going to use it for walls but then ended up building without. It is official Unity though and looks great so will try my hand at it again soon.

Commit:
Textured Walls & Coloured Floor

Made custom based on edited copyright free textures in Photoshop. Need to fix stretch on long walls
